CHANGELOG.md

```diff
@@ -7,6 +7,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.9.0] - 2024-05-07
+
+### Deprecated
+
+- This release adds a new setting `default_local_cidr_any` that defaults to
+  true to match previous behavior, but will default to false in a future
+  release. When set to false, `local_cidr` is matched correctly for firewall
+  rules on hosts acting as unsafe routers, and should be set for any firewall
+  rules you want to allow unsafe route hosts to access. See the issue and
+  example config for more details. (#1071, #1099)
+
+### Added
+
+- Nebula now has an official Docker image `nebulaoss/nebula` that is
+  distroless and contains just the `nebula` and `nebula-cert` binaries. You
+  can find it here: https://hub.docker.com/r/nebulaoss/nebula (#1037)
+
+- Experimental binaries for `loong64` are now provided. (#1003)
+
+- Added example service script for OpenRC. (#711)
+
+- The SSH daemon now supports inlined host keys. (#1054)
+
+- The SSH daemon now supports certificates with `sshd.trusted_cas`. (#1098)
+
+### Changed
+
+- Config setting `tun.unsafe_routes` is now reloadable. (#1083)
+
+- Allow `::` in `lighthouse.dns.host`. (#1115)
```

Comment:
Or "Fixed" since it was already allowed for `listen` and arguably `[::]` is _not_ a host (but rather an IPv6 host component of an IP:port combo) but I think I am only nitpicking.